docs/overview/vg_lite_tvg.rst

```diff
@@ -0,0 +1,29 @@
+.. _vg_lite_tvg:
+
+=================
+VG-Lite Simulator
+=================
+
+LVGL integrates a VG-Lite simulator based on ThorVG.
+Its purpose is to simplify the debugging of VG-Lite adaptation and reduce the time of debugging and locating problems on hardware devices.
+
+It has been integrated into the CI automated compilation and testing process to ensure that the VG-Lite rendering backend can be fully tested after each PR modification.
+
+You can find the standalone version `here <https://github.com/FASTSHIFT/vg_lite_tvg/>`_.
```

Comment:
We shouldn't link to to user repos. If needed we can add content similar to your README here.

Reply:
Done, I remove it.